### ISSUE TITLE
add support for passing arguments

### ISSUE THREAD

**Issue by woelper** [randomairborne/cargo-universal2#1]
First of all, thank you for the great crate!

I often find myself in a situation where I want to build with certain parameters, such as adding features or building without release. What are your thoughts to pass those flags to cargo universal?

**Comment by randomairborne** [comment]
Wow, I didn't expect that anyone was still using this! I used release only purely for simplicity- this application is incredibly simple, so. Adding argument support should be fairly simple- would you be willing to take a crack at it?

**Comment by woelper** [comment]
I _am_ using this, and I love the simplicity of the crate!

I went ahead and made this PR: https://github.com/randomairborne/cargo-universal2/pull/2

I am not sure you are happy with the design, as I chose to just pass on all args - as you previously defaulted to `release` builds, this will now default to `debug`. One would now have to specify `release` just as in a normal build. This may change/break functionality with users who assume cargo-universal2 produces release builds by default, but is more in line with how cargo normally operates.

Also I did not know how to test the subcommand directly - any insight would be appreciated. The way I tested was to run `cargo run -- --release` to pass extra flags - I am not sure how variables are passed with cargo subcommands.

**Comment by randomairborne** [comment]
i believe they're passed in directly. Would it be possible to check for no arguments, and then use release in that case, just for compatibility?

**Comment by woelper** [comment]
Oh, that is a neat idea! I'll do that!

**Comment by woelper** [comment]
Hmm, on second thought, there is no explicit option to pass `--debug` as far as I know, so defaulting to release means there is no way to build in debug mode. What are your thoughts? Should release be the only option?

**Comment by randomairborne** [comment]
Well, you'd only pass release if the arguments are otherwise entirely empty- so the user could pass --release if they wanted, but it's up to them if they're passing anything at all- and that will cause cargo to use debug

**Comment by clee** [comment]
~~This doesn't seem to be working for me. I'm trying `cargo universal2 --features sqlite` to build a package with sqlite support but it doesn't seem to actually be passing `--features sqlite` to the build commands.~~

EDIT: Aha, I think this is because the binary I got when I ran `cargo install universal2` didn't actually include this code. After building and installing locally, this change breaks calling the binary as `cargo universal2` (but works correctly as `cargo-universal2`).
